test/unit/resources/postgres_ident_conf_test.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Aaron Lippold, lippold@gmail.com
+# author: Rony Xavier, rx294@nyu.edu
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::PGIdentConf' do
+  describe 'PGIdentConf Paramaters' do
+    resource = load_resource('postgres_ident_conf')
+    it 'Verify postgres_ident_conf filtering by `system_username`'  do
```

Comment:
I have made the corrections and added the tests.Please review them when you get a chance. Thank you again @adamleff !